### PR TITLE
docs: Add note to readme around use multiple kubeconfigs

### PR DESCRIPTION
**Pull request by sheldonkwok**
https://github.com/kubernetes-client/javascript/issues/592

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,10 @@ Key:
   and operations using API versions that have been deprecated and removed in
   later server versions won't function correctly.
 
+# Usage Caveats
```
Comment (brendandburns) [inline README.md:133]: Change this heading to "Known Issues"

**Review by brendandburns — COMMENTED** [review]
Thanks for the PR, one minor comment.

File: README.md (hunk 1 above)
Comment (sheldonkwok) [reply README.md]: Sounds good!

**Pull request merged**